arrow-core-retrofit/build.gradle

```
@@ -7,12 +7,12 @@ apply from: "$SUB_PROJECT"
 apply from: "$DOC_CREATION"
 
 dependencies {
-    compile project(":arrow-core-data")
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
-    compile "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
-    testCompile project(":arrow-core-test")
-    testRuntime "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
-    testCompile "io.kotlintest:kotlintest-runner-junit5:$KOTLIN_TEST_VERSION", excludeArrow
-    testCompile "com.squareup.retrofit2:converter-gson:$RETROFIT_VERSION"
-    testCompile "com.squareup.okhttp3:mockwebserver:$MOCKWEBSERVER_VERSION"
+    api project(":arrow-core-data")
+    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
+    api "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
```

core-data should be included in core already; regarding retrofit, it's similar as for the kotlin-stdlib, I don't think we should impose a specific version to the users

```suggestion
    implementation project(":arrow-core-data")
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
    implementation "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
```